Automattic/wp-calypso
# Google My Business: update location connected status

This PR:
* Removes location reducer, since we have the information in site settings reducer, we should have a single source of truth.
* Hides the nudge on the Stats page when the location is connected
* Displays the Google My Business tab on the Stats page when the location is connected


#### Testing instructions
  
1. Run `git checkout update/gmb-location-connected` and start your server, or open a [live branch](https://calypso.live/?branch=update/gmb-location-connected)
2. Open the [`Stats` page](http://calypso.localhost:3000/stats/day) for eligible GMB site
3. Assert you see the nudge only while your location is not connected
![screen shot 2018-04-26 at 14 35 08](https://user-images.githubusercontent.com/326402/39303537-35093496-495f-11e8-8868-4a29758bea1b.png)
4. When location is connected, assert you see the GMB stats tab and no nudge:
![screen shot 2018-04-26 at 14 17 31](https://user-images.githubusercontent.com/326402/39303516-2307b420-495f-11e8-8f76-b2f4f19f0296.png)

  
 #### Reviews
